code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/github/GithubUtils.java

```diff
@@ -20,6 +24,25 @@ protected String accessToken() {
     }
 
     @Override
+    protected void postAuthModification(Account account) {
+        ApiRequest request = ApiContext.getContext().getApiRequest();
```

Comment:
This is to ensure that after an account is authenticated with github and rancher the api key for github that we need on subsequent requests is saved to the database. This used to happen in the token creator however it is more appropriate here.